examples/python/debug/test/_debug_example_test.py

```diff
@@ -19,7 +19,10 @@
 
 import logging
 import unittest
+import os
+import sys
 
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), ".."))
```

Comment:
Is this for manual runs? Thanks for enabling the example tests to be ran manually.

Reply:
No. This is necessary to import from the ".." directory. But I've switched from the `sys.path.insert` method to using the `imports` attribute in the build file. Should be cleaner.